idaes/generic_models/unit_models/plate_heat_exchanger.py

```diff
@@ -455,42 +456,43 @@ def rule_Gpc(blk, t):
         self.Gpc = Expression(self.flowsheet().time, rule=rule_Gpc,
                               doc='Coldside port mass velocity[kg/m2.s]')
 
-        # ----------------------------------------------------------------------
+        # ---------------------------------------------------------------------
         # Reynold & Prandtl numbers
         def rule_Re_h(blk, t, p):
             return blk.mh_in[t] * blk.channel_dia /\
                 (blk.Np[p] * blk.plate_width *
-                 blk.plate_gap * blk.hot_side.properties_in[t].visc_d)
+                 blk.plate_gap * blk.hot_fluid.properties_in[t].visc_d_phase["Liq"])
```

Comment:
This will probably need to have unit conversions to ensure consistency (unless you ensure that the spatial dimensions are consistent with the property package).

Reply:
@andrewlee94, this might be the source of the unit consistency(applies to other similar comments), I will take the first pass at converting this but let's look at it during our 10 am meeting today.

Reply:
@PTA0001 a quick check of the property packages shows that the units were accidentally set to Pa/s instead of Pa.s (line 450 in https://github.com/IDAES/idaes-pse/blob/main/idaes/power_generation/carbon_capture/mea_solvent_system/properties/MEA_solvent.py).